core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/data/Sounds.java

```diff
@@ -38,16 +44,17 @@
 	 */
 	SINGLE_CALL_STOP_BEEP("single_call_stop_beep.wav");
 
-	private String fileName = "de/unistuttgart/iaas/amyassist/amy/core/speech/data/sounds/";
+	private static final String DIRECTORY = "de/unistuttgart/iaas/amyassist/amy/core/speech/data/sounds/";
```

Comment:
this is not needed when the file are relativ the this class

Reply:
I just took that from the original implementation.